pkg/controller/remote/util_test.go

```diff
@@ -92,11 +91,4 @@ func TestDecodeKubeConfigSecret(t *testing.T) {
 			t.Fatalf("Expected decoded KubeConfig to match input")
 		}
 	})
-
-	t.Run("with invalid secret", func(t *testing.T) {
-		_, err := DecodeKubeConfigSecret(invalidSecret)
```

Comment:
Can we also remove `invalidSecret`?

Reply:
It's used in the other test `TestNewClusterClient`

Reply:
My bad - I only grepped in this file.